.github/workflows/release-prepare-hotfix.yml

```diff
@@ -21,11 +32,25 @@ jobs:
         with:
           app_id: ${{ secrets.GH_BOT_APP_ID }}
           private_key: ${{ secrets.GH_BOT_PRIVATE_KEY }}
-      - name: Checkout Repo
+
+      - name: Checkout composite actions
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: tools/actions/composites
+
+      - name: Generate ref/tag version to use during main checkout
```

Comment:
Is it really used for main checkout in this workflow? I only see format-app-tag being used when generating changeset status.

Reply:
good point, this name is wrong, will update

Reply:
updated 👍